src/utils.ts

```diff
@@ -38,3 +40,70 @@ export async function waitForStampUsable(beeDebug: BeeDebug, batchId: string): P
   }
   throw Error(`Stamp not found/usable: ${batchId}`)
 }
+
+/**
+ * Calculate usage of a given postage stamp
+ *
+ * @param stamp Postage stamp which usage should be determined
+ */
+export function getUsage({ utilization, depth, bucketDepth }: PostageBatch): number {
+  return utilization / Math.pow(2, depth - bucketDepth)
+}
+
+/**
+ * Validate if the value is an integer
+ * @param value
+ * @returns boolean
+ */
+export function isInteger(value: unknown): value is number | string | NumberString {
+  return (
+    (typeof value === 'string' && /^-?(0|[1-9][0-9]*)$/g.test(value)) ||
+    (typeof value === 'number' &&
+      value > Number.MIN_SAFE_INTEGER &&
+      value < Number.MAX_SAFE_INTEGER &&
+      Number.isInteger(value))
+  )
+}
+
+export function assertInteger(value: unknown, name = 'value'): asserts value is number | NumberString {
+  if (!isInteger(value)) throw new TypeError(`${name} is not a valid integer`)
+}
+
+export function assertDecimal(value: unknown, name = 'value'): boolean {
+  if (typeof value === 'number') {
+    value = value.toString()
+  }
+
+  if (typeof value !== 'string' || (typeof value === 'string' && !/^[-+]?[0]+\.[0-9]+$/g.test(value))) {
```

Comment:
Just leave it as is, as this will be changed with the config refactor anyhow, but btw. `2.333` is also decimal, which you do not support here, while the name does not really suggest it.